kubevirtmng/ui/DiskList.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import $ from "jquery"
+
+
+class Volume extends React.Component {
+  constructor() {
+      super();
+
+      this.state = {
+         disks: []
+      }
+
+      this.fetchData = this.fetchData.bind(this);
+    };
+
+   componentWillMount() {
+      this.fetchData();
+   };
+
+   fetchData() {
+        $.ajax({
+            type: "GET",
+            dataType: 'json',
+            url: 'http://localhost:8083/v1/volumes/'+this.props.data.name+"/",
+            success: function(response){
```

Comment:
consider indenting a little here for the nested function.

Since you're using babel, you could also use an arrow function, though it's not necessary